styleguide/derek/_partials/_footer.jade

```diff
@@ -1,2 +1,2 @@
-.container-fluid.bg-blue-gray.standard-padding
+.container-fluid.bg-blue-gray.fluid-footer-container
```

Comment:
@bighappyface thats fine with me. My only concern is on landing pages the ceiling is included and not the mainnav. In the future we're talking about including the rug on landing pages but not the rest of the site. Allowing the flexibility for including those on different "templates" and not on others would be nice.